### DISCUSSION TITLE
Adding Site Issue

### DISCUSSION THREAD

**Discussion by Memotas** [akeeba/panopticon#517] [General]
Hello,

I'm using joomla 5.0.2 with PHP 8.3.1, MySQLi 10.5.23-MariaDB, pkg_panopticon v1.0.2, Panopticon v1.0.7.

Active Ones:

Web Services - Panopticon
Web Services - Installer 
API Authentication - Web Services Joomla
User - Joomla API Token

Cron job is defined correctly.

I couldn't connect the two sites to the Akeeba Panopticon. I'm using two sites as a trial. If I send you the information, would you like to review it?

Error Message:

An error occurred loading the site extensions information.

Server error: `GET https://d----.org/api/index.php/v1/panopticon/extensions?page%5Blimit%5D=10000` resulted in a `503 Service Unavailable` response: <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd" (truncated...)

**Comment by nikosdion** [comment]
I can help you troubleshoot your issue, but I need the following connection information to your site.

**Site connection information**

* Login URL e.g. `https://www.example.com/administrator/index.php?secret123`
* Super User username
* Super User password

**(S)FTP connection information**

* Hostname
* Port (if other than 21 or 22)
* Username
* Password

Do **NOT** paste this information unencrypted. GitHub issues and discussions are publicly visible. Instead, follow the instructions below to send me an encrypted copy of the information which can only be decrypted by me:

* Go to https://keybase.io/encrypt
* In the **Recipient** field type `nikosdion`.
* It will show you my face, my name, and the fact that I have linked my same-named GitHub account; verify this information and click on my face.
* In the **Message to encrypt** area enter the information I asked you above.
* Click the Encrypt button.
* After a short while the text area changes its title to **The secret message**.
* Click in the text area to copy the secret (encrypted) message.
* Paste it to your next reply to me.

This is perfectly safe to do! It uses public key encryption, just like HTTPS, with a _much, MUCH stronger cipher_ (3744-bit Elliptic Curve Cryptography which is insanely secure). The message can only be decrypted with my secret key which is safely stored on a hardware token, protected by a PIN.

Please make sure that you test the access credentials using a separate browser / (S)FTP software than what you normally use to ensure that they really work. Please make sure that there are no geographic or other restrictions which may prevent me from accessing your site. Kindly note that I am on the [Asia/Nicosia timezone](https://www.timeanddate.com/worldclock/cyprus/nicosia) and I am handling support free of charge; some delays are to be expected.

**Comment by Memotas** [comment]
Hi,

Thank you for your interest.

I'm using the Keybase app for the first time. I sent the information you requested in encrypted form. Did you get it? Please give me information.

**Reply by brianteeman** [reply]
Click in the text area to copy the secret (encrypted) message.
Paste it to your next reply to me.

**Comment by Memotas** [comment]
-----BEGIN PGP MESSAGE-----
Version: Keybase OpenPGP v2.1.13
Comment: https://keybase.io/crypto

wcBMA6+1LLjxrhABAQf+OVYr9pXgz1kijMt0AWjtWn0TwphQQj569mUI6HY4yOT6
45mbLBviTEd042HfOeEDwwWZdh7XUJCeS30SWz4wR3DyexI6PvvkQQ2z0/4k4Gch
8gvaoHpsB0NwSCKlrv1q0gej6i4/tgsij0J0Fxwq3SqSYEjd7G71HWGxVLb92Qxr
7hL9bX7o+DgUNVF3Jx2QY9gSd92gQ4hHbqy6zqY7t8d4wkCI9MfgzhIXQuLE+ro8
W79SwUYdKCe6XZn9NrqCJucPuhU12VVoLDrpu4O3MlXhlQNHzAWwLioArjQO2Oua
Vm3rtSzX4RGklzTcN2NjlTZCVUcxcGKCznO1JbRoINLAXgFIXQWRDtK0YnrBtH1/
siF5mxPdtn6kGzr/W8oPn8vgV8190NIS0ZXzH8smHYez5PdmdBeL+Cn4JOhTyeLY
bvcCNBwylHm6fMySLfFqCcINnJyYJEvKoA1clmruNVHY5a47AbVUAaP1z01/PBmG
zQR6ZFs/OqvXDNvYOCdH3Ks6VARLTAqww8xAnSIGlLNLhQVomlvQWWo+fbgkA7rh
hvdfemNkj6TJ8udldsRwXcyWbBw0uZwTtTn/wqhQ9ayJDIJh2GJrPhXkspGuKuUD
bsKEbAwa2ALwCNPRQSiMfIrzarH49omFtu2MvoCdEEzbIE6w6YXvf5QHnKKYD3fQ
eveV8eFsItZSIJ4Dubz/fUDvM1GoztpmY655DXLM6HA=
=/hK+
-----END PGP MESSAGE-----

**Reply by nikosdion** [reply]
In your Global Configuration you have enabled Search Engine Friendly URLs but your .htaccess file does not, in fact, support Search Engine Friendly (SEF) URLs as it is completely missing _all_ of the directives shipped with Joomla! in its `htaccess.txt` site. Therefore, when Panopticon tries to access the API application using a SEF URL your server throws an error.

There are three possible fixes:

1. **Change your API endpoint**. Use the API endpoint `https://www.durutest.com/api/index.php/` in Panopticon. Note the `/index.php/` part added after `api`.
2. **Disable SEF URLs in Global Configuration**. Then you will get the correct API endpoint which, as you see, is the same as the one above.
3. **Fix SEF URLs on your site**. Copy the contents of Joomla's htaccess.txt file _above_ your current `.htaccess` contents and make any changes necessary for your server.

So, this is not a problem with Panopticon. It's a problem with setting up your site.

**Comment by Memotas** [comment]
Thank you for your effort.

I tried all three steps you mentioned. But unfortunately the problem was not solved. If you want to review, you can make every change on the sites. The sites are for testing purposes. Also, there are backups.

pkg_panopticon v1.0.3 and Panopticon v1.1.0 are installed.

**Reply by nikosdion** [reply]
These are not steps! These are **mutually exclusive** alternatives at fixing your problem.

Fix SEF URLs on your site, enabled SEF URLs in Global Configuration, and go to the Panopticon connector to see the API endpoint. Then, update your configuration in Panopticon itself and save the site.
